code/controllers/subsystem/ticker.dm

```diff
@@ -661,7 +661,9 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/yeehaw.ogg',
 		'sound/roundend/disappointed.ogg',
 		'sound/roundend/scrunglartiy.ogg',
-		'sound/roundend/petersondisappointed.ogg'\
+		'sound/roundend/hippiedisappointed.ogg',
+		'sound/roundend/heresy.ogg',
+		'sound/roundend/terminate.ogg'\
```

Comment:
Personally dislike the terminate.ogg doesn't feel up to date with 2020.
```suggestion

```

Reply:
If you are fine with removing it, you'd need to manually remove the .ogg since GitHub doesn't give me the option to suggest the deletion of a binary file.

Reply:
We could also replace leavingtg.ogg with this. (I don't have the ogg on hand.)
https://youtu.be/09s-c2JVI40